scripts/configure-noip-client

```diff
@@ -10,7 +10,7 @@ fi
 
 pushd /usr/local/src/
 
-/usr/bin/sudo /usr/bin/wget --quiet 'http://www.no-ip.com/client/linux/noip-duc-linux.tar.gz'
+/usr/bin/sudo /usr/bin/curl --location --remote-name --silent 'http://www.no-ip.com/client/linux/noip-duc-linux.tar.gz'
```

Comment:
`wget` doesn't overwrite the file if it already exists (it creates a `.0` file instead), whereas `curl` will overwrite it.